decidim-meetings/app/presenters/decidim/meetings/admin_log/meeting_presenter.rb

```diff
@@ -27,7 +27,14 @@ def diff_fields_mapping
             location: :i18n,
             location_hints: :i18n,
             start_date: :date,
-            title: :i18n
+            title: :i18n,
+            is_private: :boolean,
+            is_transparent: :boolean,
+            public_type: :string,
+            public_type_other: :string,
+            transparent_type: :string,
+            transparent_type_other: :string,
+            organizer_id: :integer
```

Comment:
This will only show the ID, which doesn't provide any info the the user checking the log. Can you make a new `ResourcePresenter` for the organizer so it's properly displayed?

You can check `NewsletterResourcePresenter` for example